### DISCUSSION TITLE
UI addon for channel sub badge + points

### DISCUSSION THREAD

**Discussion by dea9h** [Chatterino/chatterino2#4386] [Ideas]
I recommend developing a UI addon that combines the channel subscriber badge and points systems on the platform. This addon could enhance the user experience by providing users with a visual representation of their badges/badge and points. 

![image](https://user-images.githubusercontent.com/124937899/219453715-a96c2d42-16b2-4986-9c1e-79268cdbe02f.png)

**Comment by Felanbird** [comment]
We have no access to what badge you have, or how many points you have.
